### DISCUSSION TITLE
[Feature] Publish app to Rustore

### DISCUSSION THREAD

**Discussion by FreeWind6** [immich-app/immich#14743] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Rustore is an app store, an analogue of Google Play.
Your Android app is sorely missed there. Here I found the documentation: https://www.rustore.ru/help/en/developers

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Immich is already available on fdroid and through the github releases. We're not planning on adding any more app stores.
